packages/patternlab/styleguide/source/_patterns/01-atoms/09-media/picture-element.twig

```diff
@@ -1,11 +1,24 @@
-<picture class="ma__picture">
-  {% if picture.srcSets %}
-    {% for src in picture.srcSets %}
-    <source srcset="{{ src.url }}" media="(min-width: {{ src.width }}px)">
-    {% endfor %}
-  {% endif %}
+{% if picture.srcSets or picture.defaultImage %}
+  <picture class="ma__picture">
+    {% if picture.srcSets %}
+      {% for src in picture.srcSets %}
+        {% set media = "" %}
+        {% set source = src.url %}
 
-  {% if picture.defaultImage %}
-    <img src="{{ picture.defaultImage }}">
-  {% endif %}
-</picture>
+        {% if src.width is not empty %}
+          {% set media = '(min-width: ' ~ src.width ~ 'px)' %}
+        {% elseif src.mediaQuery starts with '(' %}
+          {% set media = src.mediaQuery %}
+        {% elseif src.mediaQuery ends with 'w' %}
```

Comment:
also, I think we don't need these conditions anything

Reply:
These are actually needed to cover for all the possible media query settings, I added comments exemplifying each case.